src/hipchatbot.js

```diff
@@ -0,0 +1,198 @@
+import HipchatterBase from 'hipchatter';
+import trebek from './trebek';
+import winston from 'winston';
+import request from 'request';
+import moment from 'moment';
+import App from './models/App';
+import { HIPCHAT_CALLBACK_URL } from './config';
+
+export default class Hipchatter extends HipchatterBase {
```

Comment:
The reason I did this is because this is because `get_session` is actually a proper API end point: https://www.hipchat.com/docs/apiv2/method/get_session (the snake case keeps it consistent with the hipchatter module).

BUT, I am having a bit of an issue where when the app starts up, it tries to register webhooks using the hipchatter client api, but I think maybe the module hasn't loaded all the way. I'm not quite sure... the console spits this out 3 or 4 times:

```
error:  Error: Extension not found
    at requestCallback (/var/www/html/jeopardy-bot/node_modules/hipchatter/hipchatter.js:327:32)
    at handle_output (/var/www/html/jeopardy-bot/node_modules/hipchatter/node_modules/needle/lib/needle.js:318:16)
    at Object.parsers.application/json (/var/www/html/jeopardy-bot/node_modules/hipchatter/node_modules/needle/lib/needle.js:48:5)
    at Object.Needle.response_end (/var/www/html/jeopardy-bot/node_modules/hipchatter/node_modules/needle/lib/needle.js:328:33)
    at IncomingMessage.<anonymous> (/var/www/html/jeopardy-bot/node_modules/hipchatter/node_modules/needle/lib/needle.js:272:23)
    at emitNone (events.js:72:20)
    at IncomingMessage.emit (events.js:166:7)
    at endReadableNT (_stream_readable.js:905:12)
    at doNTCallback2 (node.js:441:9)
    at process._tickDomainCallback (node.js:396:17)
```

but then eventually it starts up and logs that a valid webhook was found. 

I think when you use import it waits until the module is loaded right? How can I get that behavior here?